x-pack/legacy/plugins/monitoring/public/views/elasticsearch/nodes/index.js

```diff
@@ -39,6 +39,7 @@ uiRoutes.when('/elasticsearch/nodes', {
         }) || {};
 
       const getPageData = ($injector, _api = undefined, routeOptions = {}) => {
+        _api; // to fix eslint
```

Comment:
I think we can also use something like `// eslint-disable-next-line no-unused-vars` above the function, but this works too 👍